src/common/libeventlog/eventlogger.c

```diff
@@ -170,6 +197,7 @@ timer_cb (flux_reactor_t *r, flux_watcher_t *w, int revents, void *arg)
     if (!(f = eventlogger_commit_batch (batch->ev, batch))
         || flux_future_then (f, timeout, timer_commit_cb, batch) < 0) {
         eventlog_batch_error (batch, errno);
+        flux_future_destroy (f);
```

Comment:
this flux_future_destroy seems to be in the wrong commit?

Reply:
Oh yeah, good catch.